src/equations/shallow_water_wet_dry_2d.jl

```diff
@@ -154,26 +156,72 @@ This manufactured solution source term is specifically designed for the bottom t
 as defined in [`initial_condition_convergence_test`](@ref).
 """
 @inline function Trixi.source_terms_convergence_test(u, x, t,
-                                                     equations::ShallowWaterEquationsWetDry2D)
-    return Trixi.source_terms_convergence_test(u, x, t,
-                                               equations.basic_swe)
+                                                     equations::ShallowWaterEquations2D)
+    # Same settings as in `initial_condition_convergence_test`. Some derivative simplify because
+    # this manufactured solution velocities are taken to be constants
```

Comment:
```suggestion
    # Same settings as in `initial_condition_convergence_test`. Some derivatives simplify because
    # for this manufactured solution velocities are taken to be constants
```